lib/3scale/backend/user.rb

```diff
@@ -82,10 +82,29 @@ def self.delete!(service_id, username)
         storage.del(self.key(service_id, username))
       end
 
+      def self.delete_all(service_id)
+        service = Service.load_by_id(service_id)
+        raise UserRequiresValidService if service.nil?
+        current_scan_cursor_position = "0"
+        finished = false
+        users_set_key = service_users_set_key(service_id)
+        while !finished
+          (current_scan_cursor_position, current_usernames) =
+            storage.sscan(users_set_key, current_scan_cursor_position,
+                          { count: PIPELINED_SLICE_SIZE })
```

Comment:
Here I use the global constant named PIPELINED_SLICE_SIZE, that was already defined, but the name might be a little misleading because we are not really doing pipelining. Do you think we should create a different constant in the user.rb file to not cause confusion?

Reply:
I do

Reply:
do not create it in user.rb, since it has nothing to do with it - create a new one with comments on why is it needed in the storage module

Reply:
:+1: